spec/unit/resource_spec.rb

```diff
@@ -1256,7 +1256,7 @@ def compiled_resource_data(resource, action, err)
     end
 
     it "changes the umask in the block to the set value" do
-      resource.umask = "0123"
+      resource.umask = "0000"
```

Comment:
this looks like its just bypassing the test on unix or something, certainly doesn't look right

Reply:
I opened this PR to fix this on Windows: https://github.com/chef/chef/pull/10077